cmd/root.go

```diff
@@ -40,6 +40,7 @@ func NewRootCmd(mountFn func(config cfg.Config) error) (*cobra.Command, error) {
 and access Cloud Storage buckets as local file systems. For a technical overview
 of Cloud Storage FUSE, see https://cloud.google.com/storage/docs/gcs-fuse.`,
 		Version: getVersion(),
+		Args:    cobra.RangeArgs(1, 2),
```

Comment:
Based on manual testing, it is also counting root command as one of the argument. So the range should be 2,3

Can you also make sure that this reflects in unit test?

Reply:
Good catch! Thanks!